app/forms/registration/early_years_emails_form.rb

```diff
@@ -1,14 +1,9 @@
-module Users
+module Registration
```

Comment:
By using the name `Registration`, is the intention that we will have separate forms for user account settings?

Reply:
Have you got an example in mind? 

I think it helps group things, and yes, forms used in an alternative feature can be in a different namespace, potentially the singular feedback form. From another perspective all forms all filled by users so when or why becomes more descriptive.